### PR TITLE
Fixes for opening script files in the same window on Windows

### PR DESCRIPTION
**Pull request by lukaszunity**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs [hunk 1]
```diff
@@ -141,12 +141,12 @@ int Run (MonoDevelopOptions options)
...
-						options.NewWindow = true;
-						break;
-					}
-				}
+//				foreach (var file in startupInfo.RequestedFileList) {
+//					if (MonoDevelop.Projects.Services.ProjectService.IsWorkspaceItemFile (file.FileName)) {
+//						options.NewWindow = true;
+//						break;
+//					}
+//				}
```
Comment (Tak) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs:149]: Why not just remove this code?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StartupInfo.cs [hunk 2]
```diff
@@ -62,7 +62,7 @@ internal class StartupInfo
 		/// Matches a filename string with optional line and column 
 		/// (/foo/bar/blah.cs;22;31)
 		/// </summary>
-		public static readonly Regex FileExpression = new Regex (@"^(?<filename>[^;]+)(;(?<line>\d+))?(;(?<column>\d+))?$", RegexOptions.Compiled);
+		public static readonly Regex FileExpression = new Regex (@"^(?<filename>[^;]+)(;(?<line>-?\d+))?(;(?<column>-?\d+))?$", RegexOptions.Compiled);
```
Comment (Tak) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StartupInfo.cs:65]: Why accept negatives?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs (hunk 1 above)
Comment (lukaszunity) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs:149]: I prefer to keep it commented for reference when doing an upstream PR to fix this issue.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StartupInfo.cs (hunk 2 above)
Comment (lukaszunity) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StartupInfo.cs:65]: Unity opens files with -1 as the line number.

**Comment by Tak** [comment]
Ok, LGTM

**Pull request merged**